cron.yaml

```diff
@@ -5,3 +5,6 @@ cron:
 - description: "End-of-batch offboarding job that only runs manually"
   url: /endofbatch
   schedule: every 99999 hours
+- description: "Start of batch message to welcome people to pairing bot"
```

Comment:
This a new job that posts a welcome message to 397 Bridge. I will update both cron jobs in this file to run on a weekly cadence.